Exercise_2_Currency_Ex/exchange.md

```diff
@@ -6,16 +6,18 @@ Placing the pseudocode in this document will also allow me to keep the codebase
 
 ## Pseudocode
 
-`def estimate_value(budget, exchange_rate):`  
-`def get_change(budget, exchanging_value):`  
-`def get_value(denomination, number_of_bills):`  
-`def get_number_of_bills(budget, denomination):`  
+```Python
+def estimate_value(budget, exchange_rate):
+def get_change(budget, exchanging_value):  
```

Comment:
missed some floating back ticks on the end of this line, either that or its floating spaces, either should be removed

Reply:
Got it, for some reason the linter didn't catch it. I'll have to turn on the visibility for all characters b/c I can't see stuff like that w/o it. Thanks!

Reply:
I do the same :) I have all. whitespace visible I believe :) there's also some browser extensions fo github I use to also make that stuff easy to see :D